packages/ckeditor5-list/tests/documentlist/documentlistcommand.js

```diff
@@ -30,7 +30,11 @@ describe( 'DocumentListCommand', () => {
 		model.schema.register( 'blockQuote', { inheritAllFrom: '$container' } );
 		model.schema.extend( '$container', { allowAttributes: [ 'listType', 'listIndent', 'listItemId' ] } );
 
-		stubUid();
+		stub = stubUid();
```

Comment:
Why is this change needed? Isn't `testUtils.createSinonSandbox();` handling the restore correctly?

Reply:
Well, it didn't in documentliststylecommand, because I forgot to use this sandbox there... fixing